fodmap-project/app/meals/page.js

```diff
@@ -0,0 +1,100 @@
+"use client"
+
+import Navbar from "../shared_components/navbar";
+import MealSection from "../shared_components/meal_section";
+import styled from "styled-components";
+
+
+const StyledWrapperDiv = styled.div`
+  display: flex;
+  width: 100%;
+  background: #FCF8F3;
+  justify-content: center;
+
+  .content {
+    max-width: 1350px;
+  }
+
+  h1 {
+    font-size: 30pt;
+    font-optical-sizing: auto;
+    font-weight: 600;
+    font-style: normal;
+    margin: 30px;
+  }
+
+
+`
+
+
+export default function Meals() {
+
+  const data = [
```

Comment:
to not blow up this component, you can create some constant.js where you will keep this data.

Reply:
Done